src/BackgroundImageEditor.js

```diff
@@ -42,7 +42,12 @@ export default class RedaxtorBackgroundEditor extends Component {
             bgRepeat: this.targetDiv.style.backgroundRepeat,
             bgSize: this.targetDiv.style.backgroundSize,
             bgPosition: this.targetDiv.style.backgroundPosition,
-            alt: this.targetDiv.title || ""
+            alt: this.targetDiv.title || "",
+            pieceData: {
```

Comment:
please rename it to pieceRef and add "dataset" to data passed
dataset is user-specified data and may be usefull in abstract scenario